pkgs/tools/networking/zerotierone/default.nix

```diff
@@ -20,9 +42,22 @@ stdenv.mkDerivation rec {
       --replace 'armv5' 'armv6'
   '';
 
+  cargoDeps = rustPlatform.importCargoLock { inherit lockFile; };
```

Comment:
Please check if this works with ofborg:
```
  cargoDeps = rustPlatform.fetchCargoTarball {
    src = "${src}/zeroidc";
    name = "${pname}-${version}";
    sha256 = "sha256-PDsJtz279P2IpgiL0T92IbcANeGSUnGKhEH1dj9VtbM=";
  };
```

Reply:
it does! :+1:
(and I'd like to understand why, academically)